src/qibolab/dummy.py

```diff
@@ -255,10 +343,15 @@ def create_dummy():
     channels |= (Channel(f"drive-{i}", port=instrument[f"drive-{i}"]) for i in range(nqubits))
     channels |= (Channel(f"flux-{i}", port=instrument[f"flux-{i}"]) for i in range(nqubits))
     channels |= Channel("twpa", port=None)
+    # FIXME: Issues with the names if they are strings maybe
+    channels |= (
+        Channel(f"flux_coupler-{c}", port=instrument[f"flux_coupler-{c}"])
+        for c in itertools.chain(range(0, 2), range(3, 5))
+    )
     channels["readout"].attenuation = 0
     channels["twpa"].local_oscillator = twpa_pump
 
-    qubits, pairs = load_qubits(RUNCARD)
+    qubits, couplers, pairs = load_qubits(RUNCARD)
```

Comment:
Are we sure we want to have `couplers` separate from `qubits`?

In principle, they could be just distinguished by the specific subclass, and be listed together with other qubit objects.
If you need all and only them, you could always filter.

However, this is part of the broader question already asked: is there a reason for `Coupler`s not to be `Qubit`s?

Reply:
On the one side, the current `Qubit`s are of course way more complicate then couplings, but I wonder whether couplers will acquire their own parameters to be calibrated. For the time being, we could find a more generic name for `Qubit` and `Coupler`, and put them under a common scope (i.e. a common parent).

Reply:
Once calibrated they should have a flux value at least, maybe a operation frequency as well. I put them separate here because it made things clearer in the `create` and on the driver but the filter would also work. Also, the main reason for `Coupler`s not to be `Qubit`s is that they don't have readout and drive channels and may have different calibration we want to store for each. For the common parent I can't think of a clear suitable name for both.

Reply:
The point is that the coupler is actually a qubit on its own.

In any case, we could just name them `QuantumElement`s or something like that: if you take Ising, you could consider each spin to be a qubit, but if you take [Heisenber](https://en.wikipedia.org/wiki/Quantum_Heisenberg_model) they are not two-state systems, despite being discrete elements.

So, in this case, the abstract name would be something like *discrete elements*, that is the feature we want to highlight. But also they are those elements acting at quantum level (while outside the chip everything is classical).
Thus the proposal above.

Reply:
I will open an issue about this too https://github.com/qiboteam/qibolab/issues/609.